kodi_voice/kodi.py

```diff
@@ -479,6 +486,44 @@ def FindSong(self, heard_search):
 
     return None, None
 
+  def FindPVRChannel(self, heard_search):
+    print 'Searching for channel "%s"' % (sanitize_name(heard_search))
```

Comment:
Use heard_search.encode("utf-8") instead of sanitize_name(heard_search) here and in all other print statements.  I know we used to do this, but with the more restrictive sanitization especially, it makes debugging a bit harder since the string we see in the log might not look anything like the actual input string.